Tests/SwiftLintFrameworkTests/FileNameRuleTests.swift

```diff
@@ -50,4 +54,14 @@ class FileNameRuleTests: XCTestCase {
     func testMainDoesTriggerWithoutOverride() {
         XCTAssertEqual(try validate(fileName: "main.swift", excludedOverride: []).count, 1)
     }
+
+    func testCustomSuffixPattern() {
+        XCTAssert(try validate(fileName: "BoolExtension.swift", suffixPattern: "Extensions?").isEmpty)
+        XCTAssert(try validate(fileName: "BoolExtensions.swift", suffixPattern: "Extensions?").isEmpty)
+    }
+
+    func testCustomPrefixPattern() {
```

Comment:
can we have a test that validates a custom prefix and a custom suffix? I don't expect it to be a common case, but it'd be good to have something to catch regressions.

Reply:
What do you mean by "validate a custom prefix" etc.? I mean, we already have a test that validates that custom prefixes and suffixes work, that starts exactly here in this line, so I'm not sure what you mean...

Reply:
I meant a configuration that sets both at the same time.

Reply:
I meant a configuration that sets both at the same time.

Reply:
Ahh, I see, sure. Let me think of something ...

Reply:
Done.